src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid implements Figure {
+    private double base;
+    private double side;
+    private double height;
+    private String color;
+
+    public IsoscelesTrapezoid(double base, double side, double height, String color) {
+        this.base = base;
+        this.side = side;
+        this.height = height;
+        this.color = color;
+    }
+
+    @Override
+    public String getColor() {
+        return color;
+    }
+
+    @Override
+    public double getArea() {
+        return 0.5 * (base + side) * height;
```

Comment:
It seems like there is a mistake in the formula for the area of an isosceles trapezoid. The correct formula is 0.5 * (sum of bases) * height. Make sure that 'base' and 'side' represent the lengths of the two parallel sides of the trapezoid. If 'side' represents the length of the non-parallel sides, the formula needs to be adjusted accordingly.